pkg/k8shandler/deployment.go

```diff
@@ -343,6 +343,11 @@ func (node *deploymentNode) rollingRestart(upgradeStatus *api.ElasticsearchNodeS
 
 		if replicas > 0 {
 
+			if err := EnforceNetworkPolicy(node.self.Namespace, node.client, node.self.ObjectMeta.OwnerReferences); err != nil {
+				logrus.Warnf("Unable to create network policy: %v", err)
+				return
```

Comment:
Why is the error not return here vs L505?

Reply:
i can't remember why the `rollingRestart` function doesn't return anything but the `update` function will return an `error` but that's basic reason why this line is different than L505